### PR TITLE
[COST-3688] update Orchestrator logs

### PR DESCRIPTION
**Pull request by maskarb**
## Jira Ticket

[COST-3688](https://issues.redhat.com/browse/COST-3688)

## Description

This change updates Orchestrator logs and adds `kwargs` to log_json.

### CODE REVIEW

File: koku/masu/processor/orchestrator.py [hunk 1]
```diff
@@ -177,11 +186,11 @@ def start_manifest_processing(  # noqa: C901
         report_tasks = []
         for manifest in manifest_list:
             tracing_id = manifest.get("assembly_id", manifest.get("request_id", "no-request-id"))
-            files = manifest.get("files", [])
-            filenames = []
-            for file in files:
-                filenames.append(file.get("local_file"))
-            LOG.info(log_json(tracing_id, f"Report with manifest {tracing_id} contains the files: {filenames}"))
+            report_files = manifest.get("files", [])
+            filenames = [file.get("local_file") for file in report_files]
```
Comment (samdoran) [inline koku/masu/processor/orchestrator.py:190]: When adding lists of items to a log message, it's useful to put a cap on the number of items in the list just to make sure the message doesn't get insanely long.

File: koku/masu/processor/orchestrator.py [hunk 2]
```diff
@@ -177,11 +186,11 @@ def start_manifest_processing(  # noqa: C901
...
+            LOG.info(
```
Comment (samdoran) [inline koku/masu/processor/orchestrator.py:191]: Logging inside loops often results in spamming the logger. It's usually better to log before/after the loop.

File: koku/masu/processor/orchestrator.py [hunk 3]
```diff
@@ -234,11 +245,11 @@ def start_manifest_processing(  # noqa: C901
 
                 # this is used to create bills for previous months on GCP
                 if provider_type in [Provider.PROVIDER_GCP, Provider.PROVIDER_GCP_LOCAL]:
-                    assembly_id = manifest.get("assembly_id", None)
-                    if assembly_id:
+                    if assembly_id := manifest.get("assembly_id", None):
```
Comment (samdoran) [inline koku/masu/processor/orchestrator.py:248]:
```suggestion
                    if assembly_id := manifest.get("assembly_id"):
```

File: koku/masu/processor/orchestrator.py [hunk 4]
```diff
@@ -298,24 +309,45 @@ def prepare_monthly_report_sources(self, account, provider_uuid):
...
-        Returns:
-            (celery.result.AsyncResult) Async result for download request.
         """
+        tracing_id = provider_uuid
+        schema = account.get("schema_name")
         accounts_labeled = False
         report_months = self.get_reports(provider_uuid)
         for month in report_months:
-            LOG.info("Getting %s report files for account (provider uuid): %s", month.strftime("%B %Y"), provider_uuid)
+            LOG.info(
```
Comment (samdoran) [inline koku/masu/processor/orchestrator.py:318]: Same caution as above about logging inside loops.

File: koku/masu/processor/orchestrator.py [hunk 5]
```diff
@@ -93,14 +93,23 @@ def get_accounts(billing_source=None, provider_uuid=None):
...
-            if disable_cloud_source_processing(account.get("schema_name")) and not provider_uuid:
-                LOG.info(f"Cloud source processing disabled for {account.get('schema_name')}")
+            schema_name = account.get("schema_name")
+            if disable_cloud_source_processing(schema_name) and not provider_uuid:
+                LOG.info(log_json("get_accounts", "processing disabled for schema", schema=schema_name))
                 continue
             if disable_source(provider_uuid):
+                LOG.info(
+                    log_json(
+                        "get_accounts",
```
Comment (samdoran) [inline koku/masu/processor/orchestrator.py:103]: Is this the correct tracing ID?

Comment (maskarb) [reply koku/masu/processor/orchestrator.py:103]: There is no tracing id in this function, so I just used the function name.

File: koku/masu/processor/orchestrator.py (hunk 1 above)
Comment (maskarb) [reply koku/masu/processor/orchestrator.py:190]: I didn't change this. We've been logging the list of filenames for a while now...

**Pull request merged**